hack/copr-to-gh_pages-download.sh

```
@@ -0,0 +1,174 @@
+    find "$DEST_DIR" -name "*debuginfo*.rpm" -delete 2>/dev/null || true
+    find "$DEST_DIR" -name "*debugsource*.rpm" -delete 2>/dev/null || true
+    find "$DEST_DIR" -name "*.src.rpm" -delete 2>/dev/null || true
+
+    # Keep only flightctl-agent and flightctl-cli
+    find "$DEST_DIR" -name "*.rpm" ! -name "flightctl-agent-*.rpm" ! -name "flightctl-cli-*.rpm" -delete 2>/dev/null || true
+
+    # Create repository metadata
+    log "Creating repository metadata..."
+    for chroot_dir in "$DEST_DIR"/*; do
+        if [ -d "$chroot_dir" ]; then
+            local chroot=$(basename "$chroot_dir")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)